notebooks/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #113.        if not first_stage_model_path.exists() or not second_stage_model_path.exists():
</pre><p>If can be inverted to </p><p><code></code><code></p><p>if first_stage_model_path.exists() and second_stage_model_path.exists(): </p><p>    del ...</p><p>    return</p><p></code><code></code></p><p> </p><p>to reduce nesting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
reverted, but for me it looks less readable, maybe better to create separated functions for converting each part will be better instead